LogMonitor/src/LogMonitor/EventMonitor.cpp

```diff
@@ -436,7 +436,8 @@ EventMonitor::PrintEvent(
             status = ERROR_INVALID_HANDLE;
         }
 
-        if (ERROR_INSUFFICIENT_BUFFER == (status = GetLastError()))
+        status = GetLastError();
+        if (ERROR_INSUFFICIENT_BUFFER == status || ERROR_EVT_UNRESOLVED_VALUE_INSERT == status)

```

Comment:
`status == ERROR_INSUFFICIENT_BUFFER` instead sounds much better since the subject here is the `status`. But this is purely personal preference, so we can go with this for now.